src/View/SSTemplateParser.php

```diff
@@ -4020,10 +4020,10 @@ function CacheBlock_CacheBlockTemplate(&$res, $sub)
         // Get any condition
         $condition = isset($res['condition']) ? $res['condition'] : '';
 
```

Comment:
These changes need to be made in SSTemplateParser.peg as well, otherwise it'll get replaced later.